region/lowernorfair/east.json

```diff
@@ -6420,6 +6420,27 @@
                 }
               ]
             },
+            {
+              "id": 3,
+              "strats": [
+                {
+                  "name": "Zebbo Farm",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      {"and": [
+                        "canPauseAbuse",
+                        "Grapple"
```

Comment:
I assume we're trying to refill here, but not gaining health here is accurate for below expert, even with grapple.

Reply:
Whoops, yeah forgot the "refill" part. Added that now. The idea is that "canPauseAbuse" will go into Expert so you won't be able to logically gain health from this farm without that (or heat resistance). Aside from the precision needed to farm quickly with Grapple, the problem was that you could run out of energy while getting ammo drops, with the health bomb not being enough to reliably save you without pause abuse